makefile

```diff
@@ -0,0 +1,42 @@
+# Commake (v0.0.1) - https://niallbunting.com/commake/common/makefile/2022/03/09/commake-common-makefile.html
+
+.PHONY: all help init install build run lint test e2e plan deploy
+all: lint test run #help Full check
+
+help:
+	@echo "-- HELP --"
+	@grep '#[h]elp' makefile
+
+init: #help Run through dependencies and check
+	git secrets
+	echo "Is this node 16?"
```

Comment:
This should be node 14 for the microservices (see package.json) and https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html